### PR TITLE
Back end challenge

### PR DESCRIPTION
**Pull request by avilyn**
#### Purpose of Pull Request?
Add a backend specific challenge so our backend candidates don't have to do a front end or infrastructure challenge.
#### How should this code be validated?
Make sure the challenge is reasonable.
#### Extra Release Steps:

### CODE REVIEW

File: back_end/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Please see [films.json](films.json) for sample film data.
+
+We don't expect you to have a perfect solution in the time allotted. Do as much as is reasonable to demonstrate your technical abilities. Please submit your work by sending us a link to a GitHub repo with your code.
+
+Bonus points for the following:
+- Support pagination
+- Support filtering/sorting
+- Support for only retrieving the attributes as listed in the request
+- Document the end points (options that can be passed, etc)
+- Provide all the documentation we'd need to run the application
```
Comment (sarboc) [inline back_end/README.md:20]: I think this should be required, not a nice-to-have.

File: back_end/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Back End Challenge
+
+Your challenge is to create a simple API application using Ruby on Rails (or the web development framework of your choice). Being a SVOD service, we have film data that we need to surface through an API. Your challenge is build a small sample endpoint for retrieving film data and submitting film ratings.
```
Comment (sarboc) [inline back_end/README.md]: Missing a preposition the last sentence. Should read "Your challenge is to..." Also, we repeat "Your challenge is to" twice in this paragraph. Maybe change the paragraph to read:
```
Being a SVOD service, we have film data that we need to surface through an API. Your challenge is to create a simple API application using Ruby on Rails (or the web development framework of your choice). 
```

I think the instructions below list the two main functions we're looking for.

**Review by sarboc — CHANGES_REQUESTED** [review]
In addition to the in-line comments: tests? It looks like we don't include them in the front-end challenge. Hmmmm. I would vote for asking for them in both challenges, but I think we can leave it out if it is a skill we would be willing to teach a good developer. Or if we want to see if the dev will do them unprompted.

File: back_end/README.md (hunk 1 above)
Comment (avilyn) [reply back_end/README.md:20]: I'd rather have more code and not be able to run it than less code and documentation, personally. This is also in the bonus section in one of the other challenges. How strongly do you feel? :)

Comment (sarboc) [reply back_end/README.md:20]: Per our offline conversation, ok with leaving this as a nice-to-have.

**Review by sarboc — APPROVED** [review]

**Pull request merged**